test/unit/gateways/credorax_test.rb

```diff
@@ -209,6 +209,23 @@ def test_successful_credit
     assert response.test?
   end
 
+  def test_credit_sends_correct_action_code
+    stub_comms do
+      @gateway.credit(@amount, @credit_card)
+    end.check_request do |_endpoint, data, _headers|
+      assert_match(/O=35/, data)
+    end.respond_with(successful_credit_response)
+  end
+
+  def test_credit_sends_customer_name
```

Comment:
adding this test to show that the names end up in the credit requests

Reply:
Thanks for being so thorough 🙌